sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2504,28 +2499,28 @@
           "type": "integer"
         },
         "deviceEtag": {
-          "description": "Device's ETag",
+          "description": "TODO - which operation does this affect?",
           "type": "string"
         },
         "status": {
-          "description": "Gets the corresponding Device's Status.",
+          "description": "Gets the device's status.",
```

Comment:
The device's status? I'm not sure the property itself gets something. Some operation will get this value right?

Reply:
Good point, I'll rephrase it!